dists/gradle-aem-tailer/gradle.properties

```diff
@@ -9,4 +7,7 @@ aem.instance.local-author.httpUrl=http://localhost:4502
 aem.instance.local-publish.httpUrl=http://localhost:4503
 
 #aem.instance.int-author.httpUrl=http://x.x.x.x:4502
-#aem.instance.int-publish.httpUrl=http://y.y.y.y:4503
\ No newline at end of file
+#aem.instance.int-publish.httpUrl=http://y.y.y.y:4503
+
+aem.notificationEnabled=true
+#aem.zoneId=UTC
```

Comment:
I would keep the time zone only at the instance level, not the project level. Or change property name to:
`aem.defaultInstanceZoneId`
AEM is an abstract thing and doesn't have zoneId

Reply:
has also abstract thing like `httpurl` ; this is a perspective; GAP is a client and this properties are bound to that perspective and I am ok with that.

GAP semantics does not assume the best reflection of real world but optimizes configuration / keeping things in one place to be able to easily provide best automation possible.

Reply:
what is more ... commonly used convention for naming is [topic]_[trait] so that `instanceZoneIdDefault` instead;

however I am ok with keeping `zoneId` here. the intention is to have a place in which we say ..."I know that all my AEM instances/servers have such zoneId" .. and basing on that I can make any automation with even date calculations that could also base on that property. That's the motivation of keeping it here.